styles.css

```diff
@@ -415,4 +416,56 @@ a {
     background: #e5e5e5;
     border: 1px solid #3c3a39;
   }
+
+  .form-container {
+    background-color: #fff;
+  }
+
+  .form-container p {
+    font-family: 'Inter', sans-serif;
+    font-weight: 1000;
```

Comment:
- [ ] Please stick to the Figma design requirements as closely as possible as it is the main essence of this module. Paying attention to details is a good trait that can go a long way to satisfy your clients in your future job. 🏢 
Below is the recommended `font weight`. 

![image](https://github.com/Rebelzob/Portfolio/assets/117144028/f5e81001-e1b7-4d2e-ac6f-f4cd24932a9d)